LocalMultiplayerAgent/Config/MultiplayerSettingsValidator.cs

```diff
@@ -27,20 +27,9 @@ public MultiplayerSettingsValidator(MultiplayerSettings settings, ISystemOperati
 
             _systemOperations = systemOperations ?? SystemOperations.Default;
         }
-    
+
         public bool IsValid()
         {
-            if (string.IsNullOrWhiteSpace(_settings.OutputFolder) || string.IsNullOrWhiteSpace(_settings.TitleId))
```

Comment:
why are we removing these from here? I can see that they are used below but they are both critical so I'd rather have the code fail early.

Reply:
Ah okay I get it. It was basically because I was checking the `createDeployment` condition below, but yes I can move that up